recipes-appfw/sample-node0/files/manifest.in

```diff
@@ -0,0 +1,21 @@
+{
+    "application": "node0",
+    "description": "node0 - A NodeJS sample app.",
+    "provider": "test",
+    "groups": "test-node0",
+    "environment": {
+        "NODE0_FOO": "node0-foo",
+        "NODE0_BAR": "node0-bar",
+    },
+    "command": [
```

Comment:
@klihub yes, stop -> ExecStop.

Install and Uninstall are ways to let the application unregister something on their own, like call the server to say the device is not to be used anymore. If provided, you call this and wait a fixed amount of time (max limit) before you call `bundle-rm` or `docker rm` or whatever we end using to install the apps. Install can be provided as a nice to have, applications can infer that if they had no data, or had old data. But uninstall is basically impossible to do without explicit command.

As for the full containers, you mean full containers with shared rootfs? Because full containers with their own rootfs, they can provide that on their own. Need to think about full containers with shared rootfs...